Raptacon/Robot-2022
# 2 ball autonomous

## Types of changes


## Associated Issue(s)
Issue: #90
## Steps to Test
1. run autonomous
2. see if two ball works
3. Test transition to teleop

(line up is pointing at the ball, front bumper against tarmac line)

## Changelog
We want points

## What?
added drive sequence and changed turret turning

## Why?
so we have the equivalent of a high climb during autonomous
